.pre-commit-config.yaml

```
@@ -47,4 +47,20 @@ repos:
         language: system
         types: [python]
         pass_filenames: false # Run full test suite
-        stages: [pre-commit] # Only run on commit
+        stages: [pre-commit] # Only run on commit 
+  - repo: local
+    hooks:
+    - id: addlicense
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)